bolts/fair/models/fair_mixup.py

```diff
@@ -24,6 +24,11 @@
 __all__ = ["FairMixup"]
 
 
+class Criterion(Protocol):
+    def __call__(self, input: Tensor, target: Tensor) -> Tensor:
```

Comment:
Not sure this was necessary as there's nothing complex going on in this callable.

Reply:
it's only there for the kwargs

Reply:
But the signature doesn't include kwargs?

Reply:
as just a callable, this line was giving a type error: `criterion(input=pred, target=tgt_a)` as it didn't expect kwargs. Now it knows that input and target are the names.

Reply:
Oh I see. Sorry I thought you meant **kwargs which is one reason to use Protocol over callable.